packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -119,7 +120,7 @@ class ProductFullDetail extends Component {
             });
         }
 
-        addToCart(payload);
+        return addToCart(payload);
```

Comment:
Who is interested in this return value?

Reply:
This promise is passed to the sticky footer via `onAddToCart` and awaits a response in order to display the complete state. It could be reworked to use the redux state, probably.